From 53970ad9ee74c0f952984621472705ab94789769 Mon Sep 17 00:00:00 2001
From: Fabio Bertinatto <fbertina@redhat.com>
Date: Fri, 2 Jun 2023 11:55:31 -0300
Subject: [PATCH] UPSTREAM: <carry>: Add OpenShift tooling, images, configs and
 docs

Add CSI mock volume tests. In upstream these tests were moved
to a different package, so we stopped generating their names
in OpenShift. This patch fixes that.
---
 openshift-hack/e2e/include.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/openshift-hack/e2e/include.go b/openshift-hack/e2e/include.go
index c25fb7febf2..3275a49bfc1 100644
--- a/openshift-hack/e2e/include.go
+++ b/openshift-hack/e2e/include.go
@@ -19,4 +19,5 @@ import (
 	_ "k8s.io/kubernetes/test/e2e/node"
 	_ "k8s.io/kubernetes/test/e2e/scheduling"
 	_ "k8s.io/kubernetes/test/e2e/storage"
+	_ "k8s.io/kubernetes/test/e2e/storage/csi_mock"
 )
-- 
2.40.1

